backoffice/src/Helpers/Constants.cs

```diff
@@ -0,0 +1,6 @@
+namespace Droits.Helpers;
+
+public static class Constants
+{
+    public const string PostcodeRegex = @"^([a-zA-Z]{1,2}\d{1,2}[a-zA-Z]{0,1})\s*?(\d[a-zA-Z]{2})$";
```

Comment:
Use this in the Address form instead of hardcoded regex and good to go!

Reply:
Try:

@"(?i)^([a-z]{1,2}\d{1,2}[a-z]{0,1})\s*?(\d[a-z]{2})$"

Same as above, just shorter 

(?i) allows for case insensitivity